yt_dlp/extractor/hockeycanada.py

```diff
@@ -0,0 +1,105 @@
+import base64
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    mimetype2ext,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class HockeyCanadaIE(InfoExtractor):
+    _VALID_URL = r'https?://video\.hockeycanada\.ca/[a-z]{2}(?:-[a-z]{2})?/c/[\w-]+\.(?P<id>\d+)'
```

Comment:
I think this should be the vidflex extractor. Here just some of the sites with the same structure/backend:
```
https://myfbcgreenville.vidflex.tv/en/c/may-12th-2024.658
https://www.figureitoutbaseball.com/en/c/fiob-podcast-14-dan-bertolini-ncaa-d1-head-coach-recorded-11-29-2018.1367
https://videos.telusworldofscienceedmonton.ca/en/c/the-aurora-project-timelapse-4.577
https://www.tuffhedemantv.com/en/c/2022-tuff-hedeman-tour-hobbs-nm-january-22.227
https://www.albertalacrossetv.com/en/c/up-floor-ground-balls-one-more.3449
https://www.silenticetv.com/en/c/jp-unlocked-day-in-the-life-of-langley-ha-15u.5197
https://jphl.vidflex.tv/en/c/jp-unlocked-day-in-the-life-of-langley-ha-15u.5197
```

Reply:
Oh, but I didn't find anything about "vidflex" in this repository, maybe there is an extractor about vidflex has another name?

Reply:
No, I don't think we have an extractor for this site yet. I just meant that instead of having this be `HockeyCanadaIE` you should rename it to `VidflexIE` and also support the other sites. Just expanding the `_VALID_URL` regex should be enough.

Reply:
> you should rename it to `VidflexIE` and also support the other sites. Just expanding the `_VALID_URL` regex should be enough.

done